src/share/poudriere/common.sh

```diff
@@ -7514,6 +7517,26 @@ calculate_ospart_size() {
 	msg "OS Partiton size: ${OS_SIZE}m"
 }
 
+ingest_base_pkgs()
+{
+	# Do we have any base-port packages to ingest?
+	[ -d "${MASTERMNT}/.packages/" ] || return 0
```

Comment:
So we build ports/packages in `jail` and then copy them in during `bulk`. Why not just build them in `bulk` like normal ports? It doesn't seem the jail is even created from these packages.

The whole build inside of `jail` concerns me as it is more unsandboxed/unjailed/shared building when we should be shrinking those cases.